src/nnfusion/engine/device/cuda.cpp

```diff
@@ -49,6 +53,10 @@ CudaEngine::CudaEngine()
     g_passes->push_back(make_shared<GemmFusionPass>());
     g_passes->push_back(make_shared<BatchNormInferenceFoldingPass>());
     g_passes->push_back(make_shared<AssignLayoutPass>());
+    //superscaler pass
+    g_passes->push_back(make_shared<SuperScalerDataParallelismPass>());
+    g_passes->push_back(make_shared<GraphSerializationPass>());
```

Comment:
Does GraphSerializationPass here support the following optimization passes (e.g., kernel fusion, pattern substitution, etc) which will modify the graph?